packages/devtools_app/lib/src/inspector/flutter/story_of_your_layout/flex.dart

```diff
@@ -379,6 +393,62 @@ class _StoryOfYourFlexWidgetState extends State<StoryOfYourFlexWidget> {
         ),
       ),
     );
+
+    // cross axis does not have any empty space
+    if (crossAxisDimension(childProperties) == crossAxisDimension(properties) ||
+        crossAxisAlignment == CrossAxisAlignment.stretch) {
+      return renderChildWidget;
+    }
+
+    // cross axis have empty space so we need to visualize it
+    double emptySpaceRenderWidth, emptySpaceRenderHeight;
+    double emptySpaceWidth, emptySpaceHeight;
+    if (direction == Axis.horizontal) {
+      emptySpaceWidth = childProperties.width;
+      emptySpaceRenderWidth = renderWidth;
+
+      emptySpaceHeight = properties.height - childProperties.height;
+
+      emptySpaceRenderHeight = max(maxHeight - renderHeight, 0);
+      if (crossAxisAlignment == CrossAxisAlignment.center) {
+        emptySpaceHeight /= 2.0;
+        emptySpaceRenderHeight /= 2.0;
+      }
+    } else {
+      emptySpaceHeight = childProperties.height;
+      emptySpaceRenderHeight = renderHeight;
+
+      emptySpaceWidth = properties.width - childProperties.width;
+      emptySpaceRenderWidth = max(maxWidth - renderWidth, 0.0);
+      if (crossAxisAlignment == CrossAxisAlignment.center) {
+        emptySpaceWidth /= 2.0;
+        emptySpaceRenderWidth /= 2.0;
+      }
+    }
+    return Flex(
+      direction: properties.crossDirection,
+      children: <Widget>[
+        if (crossAxisAlignment != CrossAxisAlignment.start)
+          Expanded(
```

Comment:
these two block are identical. Add a helper method so you don't repeat the code.
E.g.
 Widget describeEmptySpace() {
    return Expanded(
            child: EmptySpaceVisualizerWidget(
              width: emptySpaceWidth,
              height: emptySpaceHeight,
              renderWidth: emptySpaceRenderWidth,
              renderHeight: emptySpaceRenderHeight,
            ),
          );
  }
 return Flex(
      direction: properties.crossDirection,
      children: <Widget>[
        if (crossAxisAlignment != CrossAxisAlignment.start)
          describeEmptySpace(),
        renderChildWidget,
        if (crossAxisAlignment != CrossAxisAlignment.end)
          describeEmptySpace(),
      ],
    );
```